framework/include/kernels/BodyForce.h

```diff
@@ -9,14 +9,7 @@
 
 #pragma once
 
-#include "Kernel.h"
-
-// Forward Declarations
-class BodyForce;
-class Function;
```

Comment:
Keep this forward declaration since you have reference to `Function` in this input file. We don't want a refactoring of includes in the future to break this file